src/operator/config/crd/k8s.otterize.com_clientintents.patched

```diff
@@ -2,7 +2,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
+    controller-gen.kubebuilder.io/version: v0.9.0
```

Comment:
These diffs indicate you ran something weird in your make process. Been there. It's not exactly well documented 🤦‍♀️ 
try: 
```
make copy-manifests-to-helm
```